config/routes/api_routes.rb

```diff
@@ -11,6 +11,7 @@ def self.extended(router)
 
       namespace :api, path: "api/:api_version", api_version: /v[.0-9]+/ do
         resource :info, only: :show, controller: "info", constraints: RouteConstraints::RegisterApiConstraint
+        resource :guide, only: :show, controller: "guide", constraints: RouteConstraints::RegisterApiConstraint
```

Comment:
I wonder whether the documentation belongs under a separate top-level path to the API itself, e.g. Publish and Apply both use `/api-docs` as the API docs home page. The docs are very different from the actual API (no authentication, HTML rather than JSON response etc.) so implementing them side-by-side doesn't help with code reuse etc.

Reply:
Should the root 'guide' page be version agnostic? (with links to the reference docs for specific versions).

Reply:
I like the idea of versioning it via the URL but it's early days and perhaps we'll move it out once we start properly documenting!